src/widget/form/profileform.cpp

```diff
@@ -118,6 +118,12 @@ ProfileForm::ProfileForm(QWidget *parent) :
     Translator::registerHandler(std::bind(&ProfileForm::retranslateUi, this), this);
 }
 
+void ProfileForm::prFileLabelUpdate()
+{
+    Nexus& nexus = Nexus::getInstance();
+    bodyUI->prFileLabel->setText(tr("Current profile: ") + nexus.getProfile()->getName() + ".tox");
```

Comment:
I would say there should be no spacebar after `profile:`, this adds confusion for traslators, some people will not translate with with spacebar, so it will look ugly.